common/djangoapps/course_modes/views.py

```diff
@@ -148,6 +150,21 @@ def get(self, request, course_id, error=None):
             "responsive": True,
             "nav_hidden": True,
         }
+
+        if enterprise_enabled():
+            enterprise_learner_data = EnterpriseApiClient().fetch_enterprise_learner_data(user=request.user)['results']
```

Comment:
I think it would be best to create a helper method to get enterprise customer data, method can be named something like `get_enterprise_customer(user=<auth_user>)` or `fetch_enterprise_cusomter(user=<auth_user>)` whichever seems better.